### ISSUE TITLE
docs-bug(COMPONENT): v17.1 (the last version to omit m3) is not present in version switcher

### ISSUE THREAD

**Issue by Kerrick** [angular/components#28787]
### Documentation Feedback

The version switcher on the docs site allows me to switch between latest and v16, but not between latest and v17.1, which is the version of Angular Material before the components began switching to m3 specs. Because v16 is not 1:1 compatible with v17, this means there's no way to view the documentation for a version that I (and probably many others) am stuck using for an indefinite period of time.

### Affected documentation page

https://material.angular.io/components/categories

**Comment by Kerrick** [comment]
See also: https://github.com/angular/components/issues/28788

**Comment by json-derulo** [comment]
Valid point, it's a bit confusing that the v17.3 docs app is already on M3 while all v17.3 projects have M2 by default.

**Comment by Kerrick** [comment]
I misunderstood that both were available; rather than making available version 17.1, it would also be fine if there was another way to switch to m2 on latest 17.x in the docs. Ideally that way would also be in the URL so one can send coworkers links to either one specifically.

**Comment by Yberion** [comment]
Yeah, it would be good to have the last `Angular 17` version of `Angular Material` with `m2`.

It would be awesome to have something like that:

- ...
- `17.3.1 (latest) M3`
- `17.3.1 (latest)`
- `v16`
- ...

Starting from `17.3.1 (latest) M3`, all new version of the doc would be m3. So for example the next version will be `18.0.0 (latest)` and it will use m3.

**Comment by szabogerArgon** [comment]
I agree.

Currently, Angular 17 is the latest LTS version, and M.2 is the default appearance when used in a project, M.3 was still experimental in Angular Material 17:
https://v17.material.angular.io/guide/material-3

This behaviour causes a lot of confusion, since our customers/UX team see the M.3 styling in the documentation for verson 17, even though M.2 is the "stable" and default styling.
